packages/conf-llvm/conf-llvm.15/opam

```diff
@@ -15,7 +15,7 @@ depexts: [
   ["llvm-15"] {os-distribution = "macports" & os = "macos"}
   ["llvm-15-dev" "zlib1g-dev" "libzstd-dev"] {os-family = "debian"}
   ["llvm15-dev"] {os-distribution = "alpine"}
-  ["llvm"] {os-family = "arch"}
+  ["llvm15"] {os-family = "arch"}
```

Comment:
These changes broke the conf-llvm package on Archlinux. Would you mind reverting those? (there is no such package, the current version of the `llvm` package is 15, there is no alias called `llvm15`)

Reply:
That's strange. I'm running Arch, and this is what I get:
```
❯ pacman -Q llvm llvm15
llvm 16.0.6-1
llvm15 15.0.7-1
```
As far as I can tell, these packages have been made available in the second half of August. As a matter of fact, I made the change because on my machine conf-llvm would complain that the LLVM version didn't match. I don't mind reverting of course, but I'd be interested to know why we end up with a different view of Arch's packages.

Reply:
Ah, interesting... I use ArchlinuxARM and it seems their repository is pretty outdated:
```
$ pacman -Q llvm llvm15
llvm 15.0.7-3
error: package 'llvm15' was not found
```
Let's leave things as-is then. Sorry for the ping

Reply:
NB: I imagine that `conf-libclang` should be modified as well? (according to `pacman`, `clang15` was release at the same time as `llvm15`)